### PR TITLE
Minor grammar/spelling improvements to the README

### PR DESCRIPTION
**Pull request by kylef**
Changes:

- Simplify installation instructions using `git clone --recursive` instead of extra submodule step
- Minor grammar/spelling improvements
- Use HTTPs for all links.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ The available options are:
 
 Name                   | Description
 ---------------------- | ----------------------------------------------------------
-`requireBlueprintName` | Require that the API Blueprint have a title (default: false)
+`requireBlueprintName` | Require parsed API Blueprints have a title (default: false)
```
Comment (pksunkara) [inline README.md]: `Required parsed API Blueprints` doesn't seem legible to me.

**Comment by pksunkara** [comment]
Just a minor thing.

File: README.md (hunk 1 above)
Comment (zdne) [inline README.md]: I'd say

API Blueprint is a format

API blueprint is a file in API Blueprint format

So: 

"Require parsed blueprints have a title"

Comment (kylef) [inline README.md]: Thanks @zdne.

This has been changed.

**Pull request merged**